src/app/csv_parser.py

```diff
@@ -0,0 +1,24 @@
+import sqlite3
+import pandas as pd
+
+from definitions import DATABASE_PATH, PLANTS_CSVTABLE_PATH
+
+if DATABASE_PATH.exists():
+    conn = sqlite3.connect(DATABASE_PATH)
+else:
+    print("No database file found")
+    exit()
+
+if PLANTS_CSVTABLE_PATH.exists():
```

Comment:
For the future:
to have the code clean, would be nice to:
- Add methods in this module instea of havin code execution in top level of the file.
- Follow single responsibility rule - My understanding of this module is having `csv_reader` -> read and parse csv file, return it in some python data structure like `dict`. Saving in db should be responsibility of a controller or some service